airflow/ui/src/pages/DagsList/DagCard.tsx

```diff
@@ -41,14 +42,20 @@ type Props = {
 const MAX_TAGS = 3;
 
 export const DagCard = ({ dag }: Props) => {
+  const navigate = useNavigate();
   const cardBorder = useColorModeValue("gray.100", "gray.700");
   const tooltipBg = useColorModeValue("gray.200", "gray.700");
 
   return (
     <Box
+      _hover={{
+        borderColor: "subtle-text",
+        cursor: "pointer",
+      }}
```

Comment:
I think we should avoid this. I know that alternatives are no straigh forward, but in the same way as we do for the table row, making `<tr>` and `<div>` clickable is not great for accessibility I believe. Can't use tab to go on that, or press 'enter' to click, screenreader will have a hard time with such pattern.

Reply:
An easy alternative is to not make the whole row clickable, but link only on the 'dag' name. (not as good but accessible).